src/validation/overseas.entity.due.diligence.validation.ts

```diff
@@ -3,11 +3,11 @@ import { body } from "express-validator";
 import { ErrorMessages } from "./error.messages";
 import { identity_address_validations } from "./fields/address.validation";
 import { VALID_CHARACTERS } from "./regex/regex.validation";
-import { identity_check_date_validations } from "./fields/date.validation";
+import { checkOptionalDate } from "./custom.validation";
 
 export const overseasEntityDueDiligence = [
 
-  ...identity_check_date_validations,
+  body("identity_date").custom((value, { req }) => checkOptionalDate(req.body["identity_date-day"], req.body["identity_date-month"], req.body["identity_date-year"])),
```

Comment:
Minor, but we've tended to put the checks immediately after the body("field-name") statement on a new indented line.